### ISSUE TITLE
iptv recording

### ISSUE THREAD

**Issue by TotalKodi** [primaeval/plugin.video.iptv.recorder#6]
hi. I'm a beginner and i wish to know how to record channales in IPTV SIMPLE CLIENT using windows and android ?

**Comment by Arias800** [comment]
You need epg for recording ^^

**Comment by primaeval** [comment]
Follow the Quick Start on the front page of this addon.

**Comment by TotalKodi** [comment]
yeah.. I do have EPG.. Is great working for me now.. but only for windows at the moment ?

**Comment by primaeval** [comment]
From the readme.md on the front page of this repo
https://github.com/primaeval/plugin.video.iptv.recorder

> This addon is an easily extensible python addon that should work with any device.
> 
> You will need a version of ffmpeg for your device.
> 
> https://ffmpeg.org/
> 
> Android builds are here: https://github.com/WritingMinds/ffmpeg-android/releases/latest
> 
> On Android this addon will copy ffmpeg to the /data/data folder so it can run.

**Comment by rmalchi** [comment]
Hi guys, thanks for this add-on! One issue for me, I've installed the ffmpeg for win 64 on my win 10 64 but when I run the recording I get an error message that Kodi can't find the ffmpeg.exe. I've configured in the recorder settings the path to the ffmpeg.exe file but still getting the can't find the ffmpeg.exe error. Any ideas how to resolve this? Thanks
